CHANGELOG.md

```diff
@@ -1,65 +1,87 @@
-Changelog
-=========
+# Changelog
+
+### 1.0.1
+
+- Adds support for the hourly query limit on the Account API endpoint
+- Fixes an issue with handling of proxies
```

Comment:
@CAMIRO I noticed you took out the "thanks" note from my suggestion. Can you please add that